cmd/deploy/deploy.go

```diff
@@ -316,34 +317,8 @@ func (dc *DeployCommand) RunDeploy(ctx context.Context, deployOptions *Options)
 		return err
 	}
 
-	// TODO: take this out to a new function deploy dependencies
```

Comment:
why the whole block of dependencies has dissapear? 🤔

Reply:
I deleted the incorrect code block 🤦 , that's why this was missing and the build was there twice. Fixed!